src/app/js/public/graph/sagas.js

```diff
@@ -83,7 +83,6 @@ export default function*() {
     yield throttle(
         500,
         [
-            LOAD_CHART_DATA,
```

Comment:
rogue change?

Reply:
This action was triggering the saga twice (here in throttle and later in a takeEvery)